addons/mail/static/src/models/composer/composer.js

```diff
@@ -291,7 +291,8 @@ function factory(dependencies) {
                 thread.loadNewMessages();
             }
             for (const threadView of this.thread.threadViews) {
-                threadView.addComponentHint('current-partner-just-posted-message', { messageId });
+                // Reset auto scroll to be able to see the newly posted message.
+                threadView.update({ hasAutoScrollOnMessageReceived: true });
```

Comment:
Potentially race condition between this and the update on scroll in the message list component.

Rarely happens in practice, and not a super big deal.
Acceptable until code in component is moved to models eventually.

Reply:
I think I had it while testing. But `throttle` on scroll is the main problem for this feature.

Another possible "manual" race condition is to scroll after just posting message and before received bus notif. In this case auto scroll will be disabled but I think the desired result (user decided to kill the scroll feature itself by scrolling again).